app/components/list-view.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  didReceiveAttrs() {
```

Comment:
Why wouldn't this be done in `didInsertElement`, this hooks seems strange to care about DOM stuff?